src/components/arrows.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+export function SampleNextArrow(props) {
+  const {
+    className, style, onClick, onKeyPressHandler,
+  } = props;
+
+  return (
+    <div
+      className={className}
+      style={{
+        ...style,
+        background: '#97BF0F',
+        width: '60px',
+        height: '40px',
+        borderRadius: '50px 0 0 50px',
+        display: 'flex',
+        justifyContent: 'left',
+        alignItems: 'center',
+        paddingLeft: '10px',
+        position: 'absolute',
+        right: '-25px',
+        zIndex: 50,
+      }}
+      onClick={onClick}
+      onKeyPress={onKeyPressHandler}
+      role="button"
+      tabIndex={0}
+    >
+      Next
+    </div>
+  );
+}
+
+export function SamplePrevArrow(props) {
+  const {
+    className, style, onClick, onKeyPressHandler,
+  } = props;
+  return (
+    <div
+      className={className}
+      style={{
```

Comment:
- [ ] It looks lke you're adding style inside the component, why not add in a separate script? 🐒